core/tests/protractor_utils/ExplorationEditorHistoryTab.js

```diff
@@ -56,6 +62,21 @@ var ExplorationEditorHistoryTab = function() {
    * Workflows
    */
 
+  this.expectCommitDatesToBeDisplayed = async function() {
+    /*
+     * This method checks if the commit dates are being displayed in
+     * List of Changes section of the history tab.
+     */
```

Comment:
I think in JS it's standard to put these kinds of function-level comments just before the function declaration

Reply:
Also, I think something's missing between `displayed in` and `List of Changes`. Maybe:

> This method checks if the commit dates are being displayed in the "List of Changes" section of the history tab.